readme.md

```diff
@@ -16,7 +16,7 @@ Enable AMP on your WordPress site, the WordPress way.
 
 ## Description ##
 
-The [AMP Project](http://amp.dev) is an open-source initiative aiming to make the web better for all. AMP enables web experiences that are consistently fast, beautiful and high-performing across distribution platforms. The official AMP plugin for WordPress supports fully integrated AMP publishing for WordPress sites, with robust capabilities and granular publisher controls.
+The [AMP Project](http://amp.dev) is an open-source initiative aiming to make the web better for all. AMP enables web experiences that are consistently fast, beautiful and high-performing across distribution platforms. AMP used to stand for “Accelerated Mobile Pages” but no longer as it is for improving web experiences regardless of the device type, whether mobile or desktop. The official AMP plugin for WordPress supports fully integrated AMP publishing for WordPress sites, with robust capabilities and granular publisher controls.
```

Comment:
Change "AMP used to stand for “Accelerated Mobile Pages” but no longer as it is for improving web experiences regardless of the device type, whether mobile or desktop." to --> "AMP formerly stood for "Accelerated Mobile Pages", but now works completely across desktop and mobile."